ecosystem/web-wallet/src/types.ts

```diff
@@ -1,6 +1,19 @@
 // Copyright (c) Aptos
 // SPDX-License-Identifier: Apache-2.0
 
+import { AptosAccount, AptosAccountObject } from 'aptos'
+
+export type AptosAccountState = AptosAccount | undefined;
+
+export interface LocalStorageState {
+  aptosAccountObject?: AptosAccountObject,
+}
+
+export const MessageMethod = {
```

Comment:
I'd prefer to make this `export type MessageMethod = 'getAccountAddress' | 'signTransaction'` if possible

Reply:
@zacharydenton do you think a readonly const object would work as well? Something like 
```
const MessageMethod = Object.freeze({
...
} as const)
```
 Or an enum type?

Reply:
I see the reasoning for adding a type if I can add it to the arg type. Currently I can't do that in a clean way because the contentscript/background script not being able to import the same modules. I will have some follow PRs this week to clean these things up. I want to get this in for the hackathon though, and then work on cleaning things up